README.md

```diff
@@ -54,15 +54,15 @@ demonstrated below:
                 "lineStyle": "dash",
             },
             {
-                "label": "Co_s(up)",
+                "label": "Co (s↑)",
```

Comment:
Hi @dou-du, I suggest you open a separated PR for this specifically.

Reply:
Ok, I will open a separated PR for Giovanni's suggestion. However, since it is just a label here, I will not change it back.